jobs/openshift4/cluster/integreatly/openshift-4-cluster-integreatly-install.yaml

```diff
@@ -0,0 +1,61 @@
+---
+- job:
+    name: openshift-4-cluster-integreatly-install
+    display-name: 'Openshift 4 Cluster Integreatly Install'
+    project-type: pipeline
+    concurrent: true
+    parameters:
+        - string:
+            name: CLUSTER_DOMAIN
+            default: 'skaletaa.intlyqe.com'
```

Comment:
Don;t think this should default to anything

Reply:
yes, default values are there just temporarily